cmd/deploy.go

```diff
@@ -153,7 +188,7 @@ The command performs the following steps:
 	deployCmd.PersistentFlags().BoolVar(&config.nobuild, "no-build", false, "Deploys the application without building a new image or modifying the deployment manifest file.")
 	deployCmd.PersistentFlags().StringVarP(&config.appDeployFile, "file", "f", "app-deploy.yaml", "The file name to use for the deployment manifest.")
 	deployCmd.PersistentFlags().BoolVar(&config.force, "force", false, "DEPRECATED - Force the reuse of the deployment manifest file if one exists.")
-	deployCmd.PersistentFlags().StringVarP(&config.namespace, "namespace", "n", "default", "Target namespace in your Kubernetes cluster")
+	deployCmd.PersistentFlags().StringVarP(&config.namespace, "namespace", "n", "", "Target namespace in your Kubernetes cluster")
```

Comment:
I see why you did this, but as a side-effect, the `appsody deploy -h` help text won't specify the default namespace anymore. Not sure if there's an easy way to avoid this though.

Reply:
Yes, I also thought about this - but I can't find another way to know if the flag was actually used...

Reply:
That's fine. Michele and I have struggled with this in the past too. Lets not worry about it now.